{
  "api": "index",
  "file": "/test/free/data_stream/190_failure_store_redirection.yml",
  "name": "Failure redirects to correct failure store when pipeline loop is detected",
  "origin": "yaml",
  "request": {
    "args": {
      "body": {
        "@timestamp": "2020-12-12",
        "foo": "bar"
      },
      "index": "logs-foobar",
      "refresh": true
    }
  },
  "response": {
    "headers": {
      "content-length": "230",
      "content-type": "application/json",
      "location": "/.fs-logs-foobar-2025.10.16-000001/_doc/a0WD7JkB6ZpyblwDc6s2",
      "x-elastic-product": "Elasticsearch"
    },
    "payload": {
      "_id": "a0WD7JkB6ZpyblwDc6s2",
      "_index": ".fs-logs-foobar-2025.10.16-000001",
      "_primary_term": 1,
      "_seq_no": 0,
      "_shards": {
        "failed": 0,
        "successful": 1,
        "total": 2
      },
      "_version": 1,
      "failure_store": "used",
      "forced_refresh": true,
      "result": "created"
    },
    "statusCode": 201
  }
}